### PR TITLE
More update modules fixes and additions

### PR DESCRIPTION
**Pull request by kacf**
This came up while working on the integration tests for update modules.

### CODE REVIEW

**Comment by kacf** [comment]
@mirzak, @lluiscampos

**Comment by mirzak** [comment]
Looks good.

File: Makefile [hunk 1]
```diff
@@ -88,14 +92,19 @@ install-modules:
 	install -m 755 -d $(prefix)$(datadir)/mender/modules/v3
 	install -m 755 $(MODULES) $(prefix)$(datadir)/mender/modules/v3/
 
+install-modules-gen:
+	install -m 755 -d $(prefix)$(bindir)
+	install -m 755 $(MODULES_ARTIFACT_GENERATORS) $(prefix)$(bindir)/
+
```
Comment (lluiscampos) [inline Makefile]: ok

I didn't commented on this originally. These scripts are not meant to be installed on the device, only in a development environment, so I thought it was a good idea to leave them outside.

Now we are adding them in the Makefile but not to the meta target `install`, so I am still ok with it :)

Comment (kacf) [reply Makefile]: :+1:

**Pull request merged**